codecparsers/nalReader.h

```diff
@@ -28,7 +28,9 @@ class NalReader : public BitReader
     NalReader(const uint8_t *data, uint32_t size);
 
     /*parse Exp-Golomb coding*/
+    bool readUe(uint32_t& v);
```

Comment:
Yeah, but changing it in all codec parser may not be completed during a short time. Could we move it as private right now to avoid similar interfaces? After all codec parser updated, we can move it as public and remove uint32_t readUe().